aztec/src/main/kotlin/org/wordpress/aztec/toolbar/AztecToolbar.kt

```diff
@@ -408,6 +411,21 @@ class AztecToolbar : FrameLayout, OnMenuItemClickListener {
         }
     }
 
+    fun isMediaModeEnabled(): Boolean {
```

Comment:
I'm not sure if this will be used in WP Android but thought it should be added in case it's needed. I can remove it if you like then add it in a separate PR if and when it's needed.